app/controllers/application_controller.rb

```diff
@@ -38,6 +60,10 @@ def default_path
     root_path
   end
 
+  def default_url_options
```

Comment:
This ends up sticking `?locale=en` etc on the end of every link the application -- is that recommended as best practice? If so I'll go with it, but my inclination would be to instead set it on the session or something instead.

Reply:
It's the behavior I noticed when dealing with Fulcrum/Heliotrope. I don't know if that's from Samvera or something the team did locally.

Reply:
Re path component approach: just dropping in the routing changes suggested by the Rails I18n guide appears to break URL generation when we use the `some_path some_model_object` convention for generating URLs. There may also be interference from `RAILS_RELATIVE_URL_ROOT`. I have a WIP to experiment with, but that's for another ticket.